core/src/test/java/org/testcontainers/utility/TestcontainersConfigurationTest.java

```diff
@@ -160,8 +160,8 @@ public void shouldReadReuseFromEnvironment() {
 
     @Test
     public void shouldTrimImageNames() {
-        userProperties.setProperty("ryuk.container.image", " testcontainersofficial/ryuk:0.3.0 ");
-        assertEquals("trailing whitespace was not removed from image name property", "testcontainersofficial/ryuk:0.3.0",newConfig().getRyukImage());
+        userProperties.setProperty("ryuk.container.image", " testcontainersofficial/ryuk:0.3.1 ");
+        assertEquals("trailing whitespace was not removed from image name property", "testcontainersofficial/ryuk:0.3.1",newConfig().getRyukImage());
```

Comment:
since we're touching this code... :)

```suggestion
        userProperties.setProperty("ryuk.container.image", " testcontainers/ryuk:0.3.1 ");
        assertEquals("trailing whitespace was not removed from image name property", "testcontainers/ryuk:0.3.1",newConfig().getRyukImage());
```